src/language/en-US.json

```diff
@@ -397,7 +397,7 @@
     "Failure_Metadata_Value_Size_Delta_Too_Large": "Validation failed because the metadata value size delta is larger in magnitude than the value size",
     "Failure_Metadata_Value_Size_Delta_Mismatch": "Validation failed because the metadata value size delta does not match expected value based on the current state",
     "Failure_Metadata_Value_Change_Irreversible": "Validation failed because a metadata value change (truncation) is irreversible",
-    "Failure_Mosaic_Invalid_Duration": "Validation failed because the duration has an invalid value",
+    "Failure_Mosaic_Invalid_Duration": "Validation failed because the duration of mosaic has an invalid value",
```

Comment:
```suggestion
    "Failure_Mosaic_Invalid_Duration": "Validation failed because the mosaic duration has an invalid value",
```

Reply:
@segfaultxavi there is already validation added to the input fields so users will know immediately anyways if they're out of range